manticore/core/smtlib/expression.py

```diff
@@ -308,6 +308,10 @@ def __ror__(self, other):
     def __invert__(self):
         return BitVecXor(self, self.cast(self.mask))
 
+    def __len__(self):
+        """Returns the bitvector size, in bytes(!)"""
+        return self.size // 8
```

Comment:
@feliam is it fine to return len of a `BitVec` in bytes?

Reply:
idk. It is a *bit* vector. 
pyZ3 does not use __len__ but rather a bv.size()

We should carefully check that it is not used for bits anywhere.

Reply:
@ehennenfent any context why do we need to add this in the first place?

Reply:
Pretty sure the original reason was to allow [this assert](https://github.com/trailofbits/manticore/blob/master/manticore/native/cpu/abstractcpu.py#L608) to pass when using a memory model that returns a BitVec.

Reply:
We need a way to read the size of a bitvec. Maybe using __len_ is not the best way. 
We already have `.size`

Are we sure we'll never support a 4 bit bitvec?

It may be better if we remove it (__len__).

Reply:
We've since found a way to make the symbolic memory model work without this workaround, so if you think it's better to delete than risk setting ourselves up for failure later, that's fine with me.

Reply:
I removed it.